### ISSUE TITLE
Aspect ratio not updated

### ISSUE THREAD

**Issue by ennoucas** [alessiomaffeis/vue-picture-input#59]
Hi !
Setting `:autoToggleAspectRatio="true"` does not update the ratio of the canvas after uploading the file.

Thanks.

**Comment by alessiomaffeis** [comment]
Hi! Do you mean it doesn't change after selecting a new image?

**Comment by ennoucas** [comment]
Yes. I was expecting that when I drop a rectangular image (not a square) it would adapt to the ratio of the image so that I can see it completely. Instead the canvas is still a square with my image cropped.  Maybe it's a bad understand of the behavior of this `autoToggleAspectRatio`

**Comment by alessiomaffeis** [comment]
It will be fixed in 3.0.1
